apps/system/js/tracking_notice.js

```diff
@@ -0,0 +1,156 @@
+/* global SystemDialog, LazyLoader, Service */
+'use strict';
+
+(function(exports) {
+
+  const TRACKING_NOTICE_KEY = 'privacy.trackingprotection.shown';
+  const LEARN_MORE_URL =
+    'https://support.mozilla.org/products/firefox-os/privacy-and-security';
+
+  /**
+   * @class TrackingNotice
+   * @param {options} object for attributes `onShow`, `onHide` callback.
+   * @extends SystemDialog
+   */
+  var TrackingNotice = function(controller) {
+    this.instanceID = 'tracking-notice';
+    this.controller = controller || {};
+    this.options = {};
+    this._settings = navigator.mozSettings;
+    var req = this._settings.createLock().get(TRACKING_NOTICE_KEY);
```

Comment:
Do we need to read the setting again here? It feels redundant to me. If we are instantiating the `TrackingNotice`, we know that it hasn't been shown yet. What's to prevent us from calling `TrackingNotice.show()` directly from `SystemDialogManager`?